avm/client/src/avmHelpers.ts

```diff
@@ -65,15 +75,15 @@ export function serializeAvmArgs(
  */
 export function deserializeAvmResult(result: any): InterpreterResult {
```

Comment:
What about changing `any` type here to specific value? I think it would be much easier for consumers to typecheck these calls.

Reply:
Same for that separate PR.